src/dataServices/dynamoDbDataService.ts

```diff
@@ -325,6 +310,30 @@ export class DynamoDbDataService implements Persistence, BulkDataAccess {
         return getExportStatusResponse;
     }
 
+    buildExportJob(initiateExportRequest: InitiateExportRequest): BulkExportJob {
+        const initialStatus: ExportJobStatus = 'in-progress';
+        const uuid = uuidv4();
+        let exportJob: BulkExportJob = {
+            jobId: uuid,
+            jobOwnerId: initiateExportRequest.requesterUserId,
+            exportType: initiateExportRequest.exportType,
+            groupId: initiateExportRequest.groupId ?? '',
+            outputFormat: initiateExportRequest.outputFormat ?? 'ndjson',
+            since: initiateExportRequest.since ?? '1800-01-01T00:00:00.000Z', // Default to a long time ago in the past
+            type: initiateExportRequest.type ?? '',
+            transactionTime: initiateExportRequest.transactionTime,
+            jobStatus: initialStatus,
+            jobFailedMessage: '',
+        };
+        if (this.enableMultiTenancy) {
+            exportJob = {
+                ...exportJob,
+                tenantId: initiateExportRequest.tenantId,
+            };

```

Comment:
I don't think you need to rebuild it

```suggestion
            exportJob.tenantId = initiateExportRequest.tenantId;
```